### DISCUSSION TITLE
Can You Access kohya_ss From Another Computer on a Local Network?

### DISCUSSION THREAD

**Discussion by poondoggle** [bmaltais/kohya_ss#314] [Q&A] [answered]
With Automatic1111 you can use --listen to accomplish this. Does kohya_ss have something similar?

**Comment by bmaltais** [comment, marked as answer]
I have added support for it in the dev branch. You can test it by issuing the following command:

```
git checkout dev
upgrade.ps1
```

To go back to the main code do:

```
git checkout master
upgrade.ps1
```

`gui.ps1 --listen "0.0.0.0"` will allow anyone to connect to the gradio webui.

**Reply by poondoggle** [reply]
> I have added support for it in the dev branch. You can test it by issuing the following command:
> 
> ```
> git checkout dev
> upgrade.ps1
> ```
> 
> To go back to the main code do:
> 
> ```
> git checkout master
> upgrade.ps1
> ```
> 
> `gui.ps1 --listen "0.0.0.0"` will allow anyone to connect to the gradio webui.

Thanks!

Will you be eventually merging into the master branch?
